src/output/plugin-loader.cpp

```diff
@@ -147,16 +148,38 @@ void plugin_manager::reload_dynamic_plugins()
     std::vector<std::string> next_plugins;
 
     auto plugin_prefix = std::string(PLUGIN_PATH "/");
+    std::vector<std::string> plugin_prefixes;
+    if (char *plugin_path = getenv("WAYFIRE_PLUGIN_PATH"))
+    {
+        std::stringstream ss (plugin_path);
+        std::string entry;
+        while (std::getline(ss, entry, ':'))
+            plugin_prefixes.push_back(entry);
+    }
+    plugin_prefixes.push_back(PLUGIN_PATH);
 
     std::string plugin_name;
     while(stream >> plugin_name)
     {
         if (plugin_name.size())
         {
             if (plugin_name.at(0) == '/')
+            {
                 next_plugins.push_back(plugin_name);
+                continue;
+            }
             else
```

Comment:
I imagine you don't want to have this `else` block anymore ...

Reply:
ping ?

Reply:
Sorry for being slow.

> I imagine you don't want to have this `else` block anymore ...

Probably not here. We should either drop it, or add the library name as a fallback after the loop through plugin_prefixes. The latter would have the effect of following back to the standard library search paths. What do you think?

Reply:
I think simply removing is fine. Searching the `/usr/lib` or whatever doesn't seem a good idea, and the code doesn't really do it - if you look closely, it searches in the default plugin path which is checked below anyway.